src/plot.jl

```diff
@@ -163,7 +163,7 @@ function plot!(plt1::Plot, plt2::Plot, plts_tail::Plot...; kw...)
             cmdidx += 1
         end
     end
-    _add_plot_title!(plt)
+    _add_plot_title!(plt, plotattributes)
```

Comment:
I'd try setting this after line 171 and fix things from there

Reply:
I think I've tried that before, but it triggers another error. I'll have a look tomorrow.

Reply:
Here is the error if you postpone `_add_plot_title!(plt)` after `_update_subplot_args(...)`. 

```julia
Error showing value of type Plots.Plot{Plots.UnicodePlotsBackend}:
ERROR: KeyError: key :yaxis not found
Stacktrace:
  [1] getindex
    @ ./dict.jl:482 [inlined]
  [2] getindex(dd::RecipesPipeline.DefaultsDict, k::Symbol)
    @ RecipesPipeline [...]/.julia/packages/RecipesPipeline/7ijBv/src/utils.jl:18
  [3] getindex(sp::Plots.Subplot{Plots.UnicodePlotsBackend}, k::Symbol)
    @ Plots [...]/Plots.jl/src/args.jl:1846
  [4] _update_min_padding!(sp::Plots.Subplot{Plots.UnicodePlotsBackend})
    @ Plots [...]/Plots.jl/src/backends.jl:108
  [5] iterate
    @ ./generator.jl:47 [inlined]
  [6] _collect(c::Matrix{AbstractLayout}, itr::Base.Generator{Matrix{AbstractLayout}, typeof(Plots._update_min_padding!)}, #unused#::Base.EltypeUnknown, isz::Base.HasShape{2})
    @ Base ./array.jl:695
  [7] collect_similar
    @ ./array.jl:606 [inlined]
  [8] map
    @ ./abstractarray.jl:2294 [inlined]
  [9] _update_min_padding!(layout::Plots.GridLayout)
    @ Plots [...]/Plots.jl/src/layouts.jl:290
 [10] prepare_output(plt::Plots.Plot{Plots.UnicodePlotsBackend})
    @ Plots [...]/Plots.jl/src/plot.jl:224
 [11] display(#unused#::Plots.PlotsDisplay, plt::Plots.Plot{Plots.UnicodePlotsBackend})
    @ Plots [...]/Plots.jl/src/output.jl:147
 [12] display(x::Any)
    @ Base.Multimedia ./multimedia.jl:328
 [13] #invokelatest#2
    @ :0 [inlined]
 [14] invokelatest
    @ :706 [inlined]
```

Reply:
I've pushed an alternative fix.